src/blockchain/compact_filters/mod.rs

```diff
@@ -467,7 +467,7 @@ pub struct BitcoinPeerConfig {
 }
 
 /// Configuration for a [`CompactFiltersBlockchain`]
-#[derive(Debug, serde::Deserialize, serde::Serialize)]
+#[derive(Debug, serde::Deserialize, serde::Serialize, Clone, PartialEq)]
```

Comment:
Looks like you also need to add `, Clone, PartialEq` to `BitcoinPeerConfig` above on line 459. Once that's in the `compact_filters` feature should build.

Reply:
oops yeah fixed.